XamarinCommunityToolkit/Effects/Touch/TouchEffect.shared.cs

```diff
@@ -0,0 +1,1208 @@
+
+		public double HoveredOpacity => GetHoveredOpacity(Control);
+
+		public double PressedOpacity => GetPressedOpacity(Control);
+
+		public double NormalScale => GetNormalScale(Control);
+
+		public double HoveredScale => GetHoveredScale(Control);
+
+		public double PressedScale => GetPressedScale(Control);
+
+		public double NormalTranslationX => GetNormalTranslationX(Control);
+
+		public double HoveredTranslationX => GetHoveredTranslationX(Control);
+
+		public double PressedTranslationX => GetPressedTranslationX(Control);
+
+		public double NormalTranslationY => GetNormalTranslationY(Control);
+
+		public double HoveredTranslationY => GetHoveredTranslationY(Control);
+
+		public double PressedTranslationY => GetPressedTranslationY(Control);
+
+		public double NormalRotation => GetNormalRotation(Control);
+
+		public double HoveredRotation => GetHoveredRotation(Control);
+
+		public double PressedRotation => GetPressedRotation(Control);
+
+		public double NormalRotationX => GetNormalRotationX(Control);
+
+		public double HoveredRotationX => GetHoveredRotationX(Control);
+
+		public double PressedRotationX => GetPressedRotationX(Control);
+
+		public double NormalRotationY => GetNormalRotationY(Control);
+
+		public double HoveredRotationY => GetHoveredRotationY(Control);
+
+		public double PressedRotationY => GetPressedRotationY(Control);
+
+		public int AnimationDuration => GetAnimationDuration(Control);
+
+		public Easing AnimationEasing => GetAnimationEasing(Control);
+
+		public int PressedAnimationDuration => GetPressedAnimationDuration(Control);
+
+		public Easing PressedAnimationEasing => GetPressedAnimationEasing(Control);
+
+		public int NormalAnimationDuration => GetNormalAnimationDuration(Control);
+
+		public Easing NormalAnimationEasing => GetNormalAnimationEasing(Control);
+
+		public int HoveredAnimationDuration => GetHoveredAnimationDuration(Control);
+
+		public Easing HoveredAnimationEasing => GetHoveredAnimationEasing(Control);
+
+		public int RippleCount => GetRippleCount(Control);
+
+		public bool? IsToggled
+		{
+			get => GetIsToggled(Control);
+			internal set => SetIsToggled(Control, value);
+		}
+
+		public ImageSource NormalBackgroundImageSource => GetNormalBackgroundImageSource(Control);
+
+		public ImageSource HoveredBackgroundImageSource => GetHoveredBackgroundImageSource(Control);
+
+		public ImageSource PressedBackgroundImageSource => GetPressedBackgroundImageSource(Control);
+
+		public Aspect NormalBackgroundImageAspect => GetNormalBackgroundImageAspect(Control);
+
+		public Aspect HoveredBackgroundImageAspect => GetHoveredBackgroundImageAspect(Control);
+
+		public Aspect PressedBackgroundImageAspect => GetPressedBackgroundImageAspect(Control);
+
+		public bool ShouldSetImageOnAnimationEnd => GetShouldSetImageOnAnimationEnd(Control);
+
+		internal bool CanExecute
+			=> IsAvailable &&
+			Control.IsEnabled &&
+			(Command?.CanExecute(CommandParameter) ?? true);
+
+		internal VisualElement Control
```

Comment:
I'm not sure about this name... On Xamarin we use `Control` to specify the `Native Control`. Since this is a `VisualElement` we suggest that we use another name, like: `Element`, `View`, etc. Thoughts?